controllers/awsmachine_controller.go

```diff
@@ -321,13 +325,29 @@ func (r *AWSMachineReconciler) reconcileNormal(ctx context.Context, machineScope
 	// TODO(vincepri): Remove this annotation when clusterctl is no longer relevant.
 	machineScope.SetAnnotation("cluster-api-provider-aws", "true")
 
+	// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-lifecycle.html
 	switch instance.State {
+	case infrav1.InstanceStatePending:
+		machineScope.V(1).Info("EC2 instance is pending", "state", instance.State, "instance-id", *machineScope.GetInstanceID())
 	case infrav1.InstanceStateRunning:
-		machineScope.Info("Machine instance is running", "instance-id", *machineScope.GetInstanceID())
+		machineScope.V(1).Info("EC2 instance is running", "state", instance.State, "instance-id", *machineScope.GetInstanceID())
 		machineScope.SetReady()
-	case infrav1.InstanceStatePending:
-		machineScope.Info("Machine instance is pending", "instance-id", *machineScope.GetInstanceID())
-	default:
+	case infrav1.InstanceStateStopping:
+		machineScope.V(1).Info("EC2 instance is stopping", "state", instance.State, "instance-id", *machineScope.GetInstanceID())
+		machineScope.SetUnready()
+	case infrav1.InstanceStateStopped:
+		machineScope.V(1).Info("EC2 instance is stopped", "state", instance.State, "instance-id", *machineScope.GetInstanceID())
+		machineScope.SetUnready()
```

Comment:
Same as below, these can be simplified in a single case condition capturing both